src/sentry/models/file.py

```diff
@@ -150,9 +150,12 @@ class Meta:
         db_table = 'sentry_file'
 
     def getfile(self, *args, **kwargs):
-        return FileObj(ChunkedFileBlobIndexWrapper(FileBlobIndex.objects.filter(
-            file=self,
-        ).select_related('blob').order_by('offset')), 'rb')
```

Comment:
This last argument is name, not mode. So the FileObj.name was being set to `rb` which was confusing.